internal/sdkprovider/service/kafkatopic/kafka_topic_cache.go

```diff
@@ -203,3 +203,11 @@ func DeleteTopicFromCache(projectName, serviceName, topicName string) {
 	}
 	t.Unlock()
 }
+
+// GetFullQueue retrieves a topics queue
+func (t *kafkaTopicCache) GetQueue(projectName, serviceName string) []string {
```

Comment:
ensure the method name accurately represents its purpose; If the function retrieves a specific type of queue or has a unique behavior, be more explicit in the name

Reply:
We have only one queue and two methods:
1. retrieves the full queue - this one `GetQueue`
2. retrieves only top 100 elements `GetQueueTop100` 

And I have comments for each of the functions describing all of that

Reply:
nit: make it more obvious by naming it `GetFullQueue`

Reply:
ok